src/include/common/types/value.h

```diff
@@ -97,25 +199,57 @@ class Value {
     std::unique_ptr<RelVal> relVal;
 };
 
+/**
+ * @brief NodeVal represents a node in the graph and stores the nodeID, name, properties of that
```

Comment:
"stores the nodeID, name, properties of that" -> "stores the nodeID, label, and properties of that"